gateway-incoming.yaml

```diff
@@ -157,6 +157,10 @@ components:
             independent results.
           items:
             $ref: '#/components/schemas/AssessmentResult'
+        status:
+          type: string
+          example: vulnerable
+          description: variable to filter on
```

Comment:
Suggestion: this doesn't really make clear what `status` means in this context. I'd recommend using the description from your nexpose-vuln-hydrator PR for clarity and consistency.

Reply:
fixed